src/loaders/jpg/tvgJpgLoader.h

```diff
@@ -47,6 +47,9 @@ class JpgLoader : public Loader
     unsigned char *image = nullptr;
     unsigned long size = 0;
     bool freeData = false;
+
+    int width, height;
```

Comment:
It looks it doesn't need additional variables.
Base class tvgLoader has default size.
    float w = 0, h = 0;         //default image size

Reply:
Sizes in tvgLoader are float. For turbojpeg sizes must be passed as integer pointers.
Additional variables could be removed, but it would look like this, that is not nice:
```
bool JpgLoader::open(const string& path) {
    ...
    int width, height;
    tjDecompressHeader3(..., &width, &height, ...);
    vw = w = width;
    vh = h = height;
    ...
}
bool JpgLoader::read() {
    ...
    int width = static_cast<int>(w);
    int height = static_cast<int>(h);
    image = (unsigned char *)tjAlloc(width * height * tjPixelSize[TJPF_BGRX]);
    ...
    tjDecompress2(..., width, 0, height, ...);
}
```

Reply:
you can use local variables and copy it to members in that function.
ex)
int width, height;
tjDecompress2(..., width, 0, height, ...);
this->width = width;
this->height = height;

Reply:
+ vw, vh, w, h must be setup in the end.
there is no necessary to allocate more memory.
static casting is not cost actually.